src/tests/suite/daos_test_common.c

```diff
@@ -1072,25 +1073,29 @@ get_server_config(char *host, char *server_config_file)
 			break;
 	}
 
-	pch = strtok(line, " ");
-	while (pch != NULL) {
-		if (strstr(pch, "--config") != NULL) {
-			if (strchr(pch, '=') != NULL)
-				strcpy(server_config_file,
-				       strchr(pch, '=') + 1);
-			else {
+	if (conf)
+		strcpy(server_config_file, DAOS_SERVER_CONF);
+	else {
+		pch = strtok(line, " ");
+		while (pch != NULL) {
+			if (strstr(pch, "--config") != NULL) {
+				if (strchr(pch, '=') != NULL)
+					strcpy(server_config_file,
+					       strchr(pch, '=') + 1);
+				else {
+					pch = strtok(NULL, " ");
+					strcpy(server_config_file, pch);
```

Comment:
It's not from this patch, but we'd pass in buffer size and use 'strncpy' to avoid buffer overflow.

Reply:
oh yes, good catch.